cohesity_management_sdk/models/cluster.py

```diff
@@ -225,7 +230,7 @@ class Cluster(object):
     }
 
     def __init__(self,
-                 apps_settings=None,
```

Comment:
This would break backward compatibility right ? or was this an optional parameter ?

Reply:
Its is an optional field.